frontend/src/components/entity/EntityMetadata.tsx

```diff
@@ -16,14 +18,25 @@ export const VideoMetadata = ({
   publicationDate,
   uploader,
   withLinks = true,
+  variant = 'wrap',
 }: {
   views?: number | null;
   publicationDate?: string | null;
   uploader?: string | null;
   withLinks?: boolean;
+  variant?: EntityMetadataVariant;
 }) => {
   const { t, i18n } = useTranslation();
 
+  let flexShrink = 1;
+  let flexWrap = 'wrap';
+
+  switch (variant) {
+    case 'uploaderOnly':
+      flexShrink = 0;
+      flexWrap = 'nowrap';
+  }
```

Comment:
Is it really more readable than:

```suggestion
  const flexWrap = variant === "wrap" ? "wrap" : "nowrap";
  const flexShrink = variant === "wrap" ? 1 : 0;
```

Reply:
The suggestion is indeed shorter and more readable. I think I was expecting to add more variants in the future.